README.md

```diff
@@ -730,6 +744,11 @@ An order for placing the rule in `pg_hba.conf`. Defaults to `150`.
 ####`target`
 This provides the target for the rule, and is generally an internal only property. Use with caution.
 
+####`postgresql_version`
+Defaults to the version set in `postgresql::server`.  Use this if you want to manage `pg_hba.conf` without managing the entire postgreSQL instance.
+
+####`manage_pg_hba_conf`
```

Comment:
I don't think this is needed here. What would be the use-case of creating a `postgresql::server::pg_hba_rule` with `manage_pg_hba_conf` set to false explicitely?